### PR TITLE
Update openstack consulting page

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update `openstack/consulting`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/openstack/consulting>
- Check that the page matches the [copy doc](https://docs.google.com/document/d/1f83OSIHVQGMjF7b0pZUKFvoubMGqi_b-OYbSpyhveX4/edit#)

## Issue / Card

Fixes [#637](https://github.com/ubuntudesign/web-squad/issues/637)

### CODE REVIEW

File: templates/openstack/consulting.html [hunk 1]
```diff
@@ -8,158 +8,253 @@
...
+        <h1>OpenStack consulting, design and delivery</h1>
+        <h2>Carrier grade, banking security, government scale. Serious private clouds are our business.</h2>
+        <p>Hundreds of successful OpenStack clouds underpin our consulting, training, architecture design, and hardware procurement advice - to help you evaluate and deliver OpenStack.</p>
+        <p>
+          <a href="/openstack/contact-us?product=foundation-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Talk to us', 'eventLabel' : 'Talk to us - Top CTA', 'eventValue' : undefined });" class="p-button--positive">Talk to us</a>
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: Engage Canonical experts

File: templates/openstack/consulting.html [hunk 2]
```diff
@@ -8,158 +8,253 @@
...
-      <div class="col-4 p-divider__block">
-        <h3>Comes with everything you need</h3>
-        <p>Foundation Cloud Builds come with all the tools and technologies required to keep your cloud operating efficiently.</p>
-      </div>
-      <div class="col-4 p-divider__block">
-        <h3>24x7 support available</h3>
-        <p>Once operational, our support plans have you covered so your cloud is maintained with guaranteed updates and upgrades for new features.</p>
-      </div>
+    <div class="row">
+      <p>Building and running an OpenStack cloud requires specialised expertise and low-level Linux OS knowledge. Canonical brings the full stack - from kernel to containers to applications - along with operating experience from hundreds of OpenStack deployments around the world to get you started efficiently.</p>
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: Missing the title "Canonical will design and deploy OpenStack on your premises"

File: templates/openstack/consulting.html [hunk 3]
```diff
@@ -8,158 +8,253 @@
...
+      <p>Once you are certain OpenStack is the right choice we can deliver your cloud in two weeks. Only then do you need to commit the devops talent required to run your cloud, or <a href="/openstack/managed">outsource that to Canonical</a> as well.</p>
     </div>
-  </section>
-
-  <section class="p-strip--light is-bordered">
     <div class="row">
-      <h2>Get started</h2>
+      <h2>From zero to private cloud - OpenStack delivery on rails</h2>
+      <p>With many years experience deploying, supporting, running and upgrading OpenStack clouds, Canonical offers a unique fixed-price private cloud delivery service with a proven reference architecture, called Foundation Cloud Build. Canonical will design your cloud together with your architects, and deploy OpenStack for you, on your premises.</p>
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: This has an extra sentence at the end. Should be: "With many years experience deploying, supporting, running and upgrading OpenStack clouds, Canonical offers a unique fixed-price private cloud delivery service with a proven reference architecture, called Foundation Cloud Build."

File: templates/openstack/consulting.html [hunk 4]
```diff
@@ -8,158 +8,253 @@
...
-      </div>
-      <div class="col-4 p-card u-align--center">
-        <a href="{{ ASSET_SERVER_URL }}8cf6fe4f-Foundation-Cloud-Build-Openstack-data-sheet.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Read the datasheet', 'eventLabel' : 'Read the datasheet - get started', 'eventValue' : undefined });"><img src="{{ ASSET_SERVER_URL }}3e9e430a-icon-documentation.svg" alt="Read the datasheet" style="max-height: 58px; margin-bottom: 12px" /></a>
-        <h4 class="u-no-margin--bottom"><a href="{{ ASSET_SERVER_URL }}8cf6fe4f-Foundation-Cloud-Build-Openstack-data-sheet.pdf"  onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Read the datasheet', 'eventLabel' : 'Read the datasheet - get started', 'eventValue' : undefined });">Read the datasheet&nbsp;&rsaquo;</a></h4>
+    <div class="row">
+      <h2>What’s available?</h2>
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: What’s available with Foundation Cloud Build?

File: templates/openstack/consulting.html [hunk 5]
```diff
@@ -8,158 +8,253 @@
...
+    <div class="row">
+      <div class="col-6">
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">Two-day onsite workshop for introductions, requirements gathering and architecture design</li>
+          <li class="p-list__item is-ticked">Existing VMware, public cloud and on-prem workload assessment</li>
+          <li class="p-list__item is-ticked">Ongoing operations automation for scaling, node replacement, upgrades</li>
+          <li class="p-list__item is-ticked">Full high availability of all OpenStack control plane and API services</li>
+          <li class="p-list__item is-ticked">Containerised, hyper-converged or discrete architecture flexibility</li>
+          <li class="p-list__item is-ticked">Optimised KVM hypervisor for virtual machines and LXD container guests for ultra-dense Linux</li>
+          <li class="p-list__item is-ticked">Resilient network-based Ceph and direct-attach storage options for guests</li>
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: The bullet "VMware migration tools and services" should be in this column

File: templates/openstack/consulting.html [hunk 6]
```diff
@@ -8,158 +8,253 @@
...
-        <li class="p-list__item is-ticked">Includes tools for ongoing operations management</li>
-        <li class="p-list__item is-ticked">KVM and LXD hypervisors</li>
-        <li class="p-list__item is-ticked">Ceph-based storage</li>
-        <li class="p-list__item is-ticked">Full HA implementation of all control services in converged or discrete architecture</li>
-        <li class="p-list__item is-ticked">Automation and monitoring packages such as Juju, Prometheus, Nagios and more are included as standard</li>
-        <li class="p-list__item is-ticked">Integrated log management with Elastic, Logstash, and Kibana</li>
-        <li class="p-list__item is-ticked">Complete handover with tenant onboarding, documentation, and details of ongoing support</li>
-      </ul>
+      <p>If you have complex requirements, Canonical structures the engagement as a series of iterations, starting with proven and well-understood components and architecture. The goal is rapid engagement with other parts of your business that you hope to win as internal customers, and evolution to add capabilities based on their feedback together with your requirements as the IT infrastructure owners.</p>
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: MIssing title "Working with Canonical"

Comment (anthonydillon) [reply templates/openstack/consulting.html]: Also this sentence is missing a link

File: templates/openstack/consulting.html [hunk 7]
```diff
@@ -8,158 +8,253 @@
...
+              <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}9b252f48-telenet-logo-png-transparent.png" alt="Telnet" />
+            </li>
+          </ul>
+        </div>
       </div>
+    </section>
+
+  <section class="p-strip--light is-bordered">
+    <div class="row">
+      <h2>Get started on your OpenStack journey with Canonical, the global leaders</h2>
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: This section has moved up

File: templates/openstack/consulting.html [hunk 8]
```diff
@@ -8,158 +8,253 @@
...
+      <p>** Nexenta, Solidfire, Quobyte, ScaleIO and more available as add-ons</p>
+      <p><a href="/openstack/contact-us?product=foundation-cloud">For more information about services please contact us&nbsp;&rsaquo;</a></p>
     </div>
   </section>
 
-  <section class="p-strip--light is-bordered">
+  <section class="p-strip is-bordered">
     <div class="row">
-      <h2>Pricing</h2>
+      <h2>Support and operations - for smooth running clouds, 24/7</h2>
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: Please make the dash a `&mdash;`

File: templates/openstack/consulting.html [hunk 9]
```diff
@@ -8,158 +8,253 @@
...
     </div>
     <div class="row">
-      <div class="col-10">
-        {% include "shared/pricing/_openstack-consulting.html" %}
+      <div class="col-6">
+        <h3>Ubuntu Advantage - Enterprise Support</h3>
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: This should be on two lines

File: templates/openstack/consulting.html [hunk 10]
```diff
@@ -8,158 +8,253 @@
...
+        </ul>
+        <p>$1,500 per server per year</p>
+        <p><a href="/support">More about Ubuntu Advantage&nbsp;&rsaquo;</a></p>
       </div>
-    </div>
-    <div class="row">
-      <div class="col-8">
-        <p>For more information about pricing please <a href="/openstack/contact-us?product=foundation-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'For more information about pricing please contact us', 'eventLabel' : 'For more information about pricing please contact us - pricing cta', 'eventValue' : undefined });">contact us&nbsp;&rsaquo;</a></p>
+      <div class="col-6">
+        <h3>Fully Managed OpenStack</h3>
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: Missing bootstack title

**Comment by anasereijo** [comment]
@steverydz 

screen shot 2018-06-20 at 17 18 00

screen shot 2018-06-21 at 10 50 06

screen shot 2018-06-21 at 10 50 13

screen shot 2018-06-21 at 10 56 25

screen shot 2018-06-21 at 11 00 08

**Comment by frankiegranato** [comment]
Comments in the copydoc for you @steverydz

**Comment by steverydz** [comment]
@ant @anasereijo @frankiegranato Changes made. Ready for another review

**Comment by anasereijo** [comment]
@steverydz 
I think you forgot to amend this one:
image

and please amend this:
screen shot 2018-06-21 at 15 31 07

can we please add an image here? (link on copy doc)
screen shot 2018-06-21 at 15 37 23

**Comment by steverydz** [comment]
@anasereijo Changes made.

**Comment by frankiegranato** [comment]
See copy doc for amendments to the logos we have permissions to use.  
Basically - we need to use LG (liberty Global) instead of telnet and drop yahoo. 

To be extra clear: We must not allow any telnet or Yahoo Japan logos appear on the site.

**Comment by anasereijo** [comment]
@steverydz please amend this section:

screen shot 2018-06-21 at 18 00 05

**Comment by steverydz** [comment]
@anasereijo @frankiegranato Done

File: templates/openstack/consulting.html [hunk 11]
```diff
@@ -8,68 +8,218 @@
...
     </div>
     <div class="row u-equal-height">
       <div class="col-4 p-card u-align--center">
-        <a href="/openstack/contact-us?product=foundation-cloud"  onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Talk to us', 'eventLabel' : 'Talk to us - get started', 'eventValue' : undefined });"><img src="{{ ASSET_SERVER_URL }}d6146a84-icon-talk-to-us.svg" alt="Talk to us" style="max-height: 58px; margin-bottom: 12px" /></a>
-        <h4 class="u-no-margin--bottom"><a href="/openstack/contact-us?product=foundation-cloud"  onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Talk to us', 'eventLabel' : 'Talk to us - get started', 'eventValue' : undefined });">Talk to us&nbsp;&rsaquo;</a></h4>
+        <a href="/cloud/contact-us?product=foundation-cloud"  onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Talk to us', 'eventLabel' : 'Talk to us - get started', 'eventValue' : undefined });"><img class="p-card__icon" src="{{ ASSET_SERVER_URL }}d6146a84-icon-talk-to-us.svg" alt="Talk to us" /></a>
```
Comment (pmahnke) [inline templates/openstack/consulting.html]: should link to /openstack/contactus?product=foundation-cloud

File: templates/openstack/consulting.html [hunk 12]
```diff
@@ -8,68 +8,218 @@
...
+        <h1>OpenStack consulting, design and delivery</h1>
+        <h2 class="p-heading--three">Carrier grade, banking security, government scale. Serious private clouds are our business.</h2>
+        <p>Hundreds of successful OpenStack clouds underpin our consulting, training, architecture design, and hardware procurement advice - to help you evaluate and deliver OpenStack.</p>
+        <p>
+          <a href="/openstack/contact-us?product=foundation-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Engage Canonical exports', 'eventLabel' : 'Engage Canonical exports - Top CTA', 'eventValue' : undefined });" class="p-button--positive">Engage Canonical experts</a>
+          &#8195;
+          <a href="/openstack/canonical-openstack-consulting.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Read our consulting datasheet', 'eventLabel' : 'Read our consulting datasheet - Top CTA', 'eventValue' : undefined });" class="p-button--neutral">Read our consulting datasheet</a>
```
Comment (pmahnke) [inline templates/openstack/consulting.html]: should be https://assets.ubuntu.com/v1/8cf6fe4f-Foundation-Cloud-Build-Openstack-data-sheet.pdf for now

**Comment by anasereijo** [comment]
looking good @steverydz just 3 tiny changes and it's ready

screen shot 2018-06-22 at 11 59 07

screen shot 2018-06-22 at 12 00 03

screen shot 2018-06-22 at 12 00 47

File: templates/openstack/consulting.html [hunk 13]
```diff
@@ -8,68 +8,218 @@
...
+    <div class="row">
+      <h2>Canonical will design and deploy OpenStack on your premises</h2>
+    </div>
+    <div class="row">
+      <div class="u-equal-height">
+        <div class="col-8">
+          <p>Building and running an OpenStack cloud requires specialised expertise and low-level Linux OS knowledge. Canonical brings the full stack &mdash; from kernel to containers to applications - along with operating experience from hundreds of OpenStack deployments around the world to get you started efficiently.</p>
+          <p>Once you are certain OpenStack is the right choice we can deliver your cloud in two weeks. Only then do you need to commit the devops talent required to run your cloud, or <a href="/openstack/managed">outsource that to Canonical</a> as well.</p>
+        </div>
+        <div class="col-4 u-hide--small u-vertically-center">
```
Comment (pmahnke) [inline templates/openstack/consulting.html]: personally, I would add 'prefix-1' and make the illustration smaller

image

**Comment by steverydz** [comment]
@anasereijo Done

File: templates/openstack/consulting.html (hunk 13 above)
Comment (steverydz) [reply templates/openstack/consulting.html]: I set a max-width on it as @anasereijo requested it so this shouldn't be necessary now

File: templates/openstack/consulting.html [hunk 14]
```diff
@@ -8,68 +8,218 @@
...
+          </tr>
+          <tr>
+            <td>On-site workshop and training</td>
+            <td>2 days</td>
+            <td>4 days</td>
+          </tr>
+          <tr>
+            <td>Hypervisor options</td>
+            <td>KVM and LXD</td>
+            <td>KVM, LXD, Hyper-V</td>
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: KVM, LXD and Hyper-V

File: templates/openstack/consulting.html [hunk 15]
```diff
@@ -101,65 +248,32 @@ <h2 class="p-muted-heading u-align--center">Companies using OpenStack with Canon
...
-            <li class="p-list__item is-ticked">We take full operational control  of your OpenStack cloud</li>
-            <li class="p-list__item is-ticked">Clear SLA and regular independent audits for quality assurance</li>
-            <li class="p-list__item is-ticked">If you opt to take over operations, we will jointly plan a smooth transition including training and knowledge transfer</li>
-            <p><a href="/openstack/managed">Find out more about BootStack&nbsp;&rsaquo;</a></p>
-          </ul>
-        </div>
-      </div>
-    </div>
+  <section class="p-strip--light is-deep is-bordered">
+    {% include "shared/_resources_openstack.html"%}
```
Comment (anthonydillon) [inline templates/openstack/consulting.html]: This is not the same but not sure if the copy docs is correct or not

**Pull request merged**